friendly-captcha/public/widgets.php

```diff
@@ -89,5 +93,21 @@ function frcaptcha_generate_widget_tag($sitekey, $language, $extra_attributes =
     $extra_attributes);
 }
 
+$frcaptcha_skip_style_injection_tag_injected = false;
```

Comment:
Just for me to learn: is this how one would do this in PHP? I don't know about the event model or when wat code runs exactly and how often. Does this file gets run top to bottom on every file?

If not I can imagine that this causes issues with concurrent requests?

Reply:
As far as I know each request would starts a new "instance" of the PHP script. So values stored in global variables are not shared between requests.
If there is a better way to inject an element only once per page, I'm happy to make use of it 😅